### DISCUSSION TITLE
Bun v1.1.0

### DISCUSSION THREAD

**Discussion by Jarred-Sumner** [oven-sh/bun#9776] [Announcements]
To install Bun v1.1:
```bash
curl https://bun.sh/install | bash
# or you can use npm
# npm install -g bun
```

To upgrade to Bun v1.1:
```bash
bun upgrade
```

### Thank you to 364 contributors since Bun 1.0!
- [@0x346e3730](https://github.com/0x346e3730)
- [@0xflotus](https://github.com/0xflotus)
- [@2hu12](https://github.com/2hu12)
- [@52](https://github.com/52)
- [@7f8ddd](https://github.com/7f8ddd)
- [@A-D-E-A](https://github.com/A-D-E-A)
- [@a4addel](https://github.com/a4addel)
- [@Aarav-Juneja](https://github.com/Aarav-Juneja)
- [@AaronDewes](https://github.com/AaronDewes)
- [@aarvinr](https://github.com/aarvinr)
- [@aayushbtw](https://github.com/aayushbtw)
- [@adrienbrault](https://github.com/adrienbrault)
- [@adtac](https://github.com/adtac)
- [@akash1412](https://github.com/akash1412)
- [@akumarujon](https://github.com/akumarujon)
- [@alangecker](https://github.com/alangecker)
- [@alexandertrefz](https://github.com/alexandertrefz)
- [@alexkates](https://github.com/alexkates)
- [@alexlamsl](https://github.com/alexlamsl)
- [@almmiko](https://github.com/almmiko)
- [@amartin96](https://github.com/amartin96)
- [@amt8u](https://github.com/amt8u)
- [@andyexeter](https://github.com/andyexeter)
- [@annervisser](https://github.com/annervisser)
- [@antongolub](https://github.com/antongolub)
- [@aquapi](https://github.com/aquapi)
- [@aralroca](https://github.com/aralroca)
- [@Arden144](https://github.com/Arden144)
- [@argosphil](https://github.com/argosphil)
- [@ArnaudBarre](https://github.com/ArnaudBarre)
- [@arturovt](https://github.com/arturovt)
- [@ashoener](https://github.com/ashoener)
- [@asilvas](https://github.com/asilvas)
- [@asomethings](https://github.com/asomethings)
- [@aszenz](https://github.com/aszenz)
- [@audothomas](https://github.com/audothomas)
- [@AugusDogus](https://github.com/AugusDogus)
- [@AvantaR](https://github.com/AvantaR)
- [@axlEscalada](https://github.com/axlEscalada)
- [@babarkhuroo](https://github.com/babarkhuroo)
- [@baboon-king](https://github.com/baboon-king)
- [@bdenham](https://github.com/bdenham)
- [@BeeMargarida](https://github.com/BeeMargarida)
- [@benjervis](https://github.com/benjervis)
- [@BeyondMagic](https://github.com/BeyondMagic)
- [@bh1337x](https://github.com/bh1337x)
- [@birkskyum](https://github.com/birkskyum)
- [@bjon](https://github.com/bjon)
- [@blimmer](https://github.com/blimmer)
- [@booniepepper](https://github.com/booniepepper)
- [@brablc](https://github.com/brablc)
- [@bradymadden97](https://github.com/bradymadden97)
- [@brettgoulder](https://github.com/brettgoulder)
- [@brianknight10](https://github.com/brianknight10)
- [@BrookJeynes](https://github.com/BrookJeynes)
- [@brookslybrand](https://github.com/brookslybrand)
- [@Brooooooklyn](https://github.com/Brooooooklyn)
- [@browner12](https://github.com/browner12)
- [@bru02](https://github.com/bru02)
- [@buffaybu](https://github.com/buffaybu)
- [@buhrmi](https://github.com/buhrmi)
- [@Cadienvan](https://github.com/Cadienvan)
- [@camero2734](https://github.com/camero2734)
- [@capaj](https://github.com/capaj)
- [@cdfzo](https://github.com/cdfzo)
- [@cena-ko](https://github.com/cena-ko)
- [@cfal](https://github.com/cfal)
- [@CGQAQ](https://github.com/CGQAQ)
- [@chawyehsu](https://github.com/chawyehsu)
- [@chocolateboy](https://github.com/chocolateboy)
- [@chrisbodhi](https://github.com/chrisbodhi)
- [@chrishutchinson](https://github.com/chrishutchinson)
- [@ciceropablo](https://github.com/ciceropablo)
- [@Cilooth](https://github.com/Cilooth)
- [@clay-curry](https://github.com/clay-curry)
- [@codehz](https://github.com/codehz)
- [@colinhacks](https://github.com/colinhacks)
- [@Connormiha](https://github.com/Connormiha)
- [@coratgerl](https://github.com/coratgerl)
- [@cornedor](https://github.com/cornedor)
- [@CyberFlameGO](https://github.com/CyberFlameGO)
- [@cyfdecyf](https://github.com/cyfdecyf)
- [@cyfung1031](https://github.com/cyfung1031)
- [@DaleSeo](https://github.com/DaleSeo)
- [@danadajian](https://github.com/danadajian)
- [@DarthDanAmesh](https://github.com/DarthDanAmesh)
- [@davidmhewitt](https://github.com/davidmhewitt)
- [@davlgd](https://github.com/davlgd)
- [@Dawntraoz](https://github.com/Dawntraoz)
- [@desm](https://github.com/desm)
- [@DevinJohw](https://github.com/DevinJohw)
- [@dfabulich](https://github.com/dfabulich)
- [@dfaio](https://github.com/dfaio)
- [@Didas-git](https://github.com/Didas-git)
- [@diogo405](https://github.com/diogo405)
- [@dmitri-gb](https://github.com/dmitri-gb)
- [@DontBreakAlex](https://github.com/DontBreakAlex)
- [@dotspencer](https://github.com/dotspencer)
- [@dottedmag](https://github.com/dottedmag)
- [@DuGlaser](https://github.com/DuGlaser)
- [@dylang](https://github.com/dylang)
- [@e253](https://github.com/e253)
- [@ebidel](https://github.com/ebidel)
- [@eduardvercaemer](https://github.com/eduardvercaemer)
- [@eemelipa](https://github.com/eemelipa)
- [@eknowles](https://github.com/eknowles)
- [@EladBezalel](https://github.com/EladBezalel)
- [@eliot-akira](https://github.com/eliot-akira)
- [@emlez](https://github.com/emlez)
- [@eriklangille](https://github.com/eriklangille)
- [@ErikOnBike](https://github.com/ErikOnBike)
- [@eroblaze](https://github.com/eroblaze)
- [@eventualbuddha](https://github.com/eventualbuddha)
- [@fdb](https://github.com/fdb)
- [@fecony](https://github.com/fecony)
- [@fehnomenal](https://github.com/fehnomenal)
- [@FireSquid6](https://github.com/FireSquid6)
- [@fmajestic](https://github.com/fmajestic)
- [@fneco](https://github.com/fneco)
- [@FortyGazelle700](https://github.com/FortyGazelle700)
- [@G-Rath](https://github.com/G-Rath)
- [@gabry-ts](https://github.com/gabry-ts)
- [@gamedevsam](https://github.com/gamedevsam)
- [@gaurishhs](https://github.com/gaurishhs)
- [@ggobbe](https://github.com/ggobbe)
- [@gnuns](https://github.com/gnuns)
- [@gtramontina](https://github.com/gtramontina)
- [@guarner8](https://github.com/guarner8)
- [@guest271314](https://github.com/guest271314)
- [@h2210316651](https://github.com/h2210316651)
- [@Hamcker](https://github.com/Hamcker)
- [@Hanaasagi](https://github.com/Hanaasagi)
- [@hborchardt](https://github.com/hborchardt)
- [@HForGames](https://github.com/HForGames)
- [@hiadamk](https://github.com/hiadamk)
- [@HK-SHAO](https://github.com/HK-SHAO)
- [@hugo-syn](https://github.com/hugo-syn)
- [@huseeiin](https://github.com/huseeiin)
- [@hustLer2k](https://github.com/hustLer2k)
- [@I-A-S](https://github.com/I-A-S)
- [@igorshapiro](https://github.com/igorshapiro)
- [@igorwessel](https://github.com/igorwessel)
- [@iidebyo](https://github.com/iidebyo)
- [@Illyism](https://github.com/Illyism)
- [@ImBIOS](https://github.com/ImBIOS)
- [@imcatwhocode](https://github.com/imcatwhocode)
- [@ImLunaHey](https://github.com/ImLunaHey)
- [@jaas666](https://github.com/jaas666)
- [@jakeboone02](https://github.com/jakeboone02)
- [@jakeg](https://github.com/jakeg)
- [@james-elicx](https://github.com/james-elicx)
- [@jamesgordo](https://github.com/jamesgordo)
- [@jasperkelder](https://github.com/jasperkelder)
- [@jcarpe](https://github.com/jcarpe)
- [@jcbhmr](https://github.com/jcbhmr)
- [@jecquas](https://github.com/jecquas)
- [@JeremyFunk](https://github.com/JeremyFunk)
- [@jeroenpg](https://github.com/jeroenpg)
- [@jeroenvanrensen](https://github.com/jeroenvanrensen)
- [@jerome-benoit](https://github.com/jerome-benoit)
- [@jhmaster2000](https://github.com/jhmaster2000)
- [@JibranKalia](https://github.com/JibranKalia)
- [@JoaoAlisson](https://github.com/JoaoAlisson)
- [@joeyw](https://github.com/joeyw)
- [@johnpyp](https://github.com/johnpyp)
- [@jonahsnider](https://github.com/jonahsnider)
- [@jonathantneal](https://github.com/jonathantneal)
- [@JorgeJimenez15](https://github.com/JorgeJimenez15)
- [@joseph082](https://github.com/joseph082)
- [@jrz](https://github.com/jrz)
- [@jsparkdev](https://github.com/jsparkdev)
- [@jt3k](https://github.com/jt3k)
- [@jumoog](https://github.com/jumoog)
- [@kaioduarte](https://github.com/kaioduarte)
- [@kantuni](https://github.com/kantuni)
- [@karlbohlmark](https://github.com/karlbohlmark)
- [@karmabadger](https://github.com/karmabadger)
- [@keepdying](https://github.com/keepdying)
- [@kingofdreams777](https://github.com/kingofdreams777)
- [@kitsuned](https://github.com/kitsuned)
- [@klatka](https://github.com/klatka)
- [@knightspore](https://github.com/knightspore)
- [@kosperera](https://github.com/kosperera)
- [@kpracuk](https://github.com/kpracuk)
- [@krk](https://github.com/krk)
- [@kryparnold](https://github.com/kryparnold)
- [@kucukkanat](https://github.com/kucukkanat)
- [@kunokareal](https://github.com/kunokareal)
- [@kyr0](https://github.com/kyr0)
- [@L422Y](https://github.com/L422Y)
- [@LapsTimeOFF](https://github.com/LapsTimeOFF)
- [@lei-rs](https://github.com/lei-rs)
- [@lgarron](https://github.com/lgarron)
- [@lino-levan](https://github.com/lino-levan)
- [@lithdew](https://github.com/lithdew)
- [@liz3](https://github.com/liz3)
- [@Longju000](https://github.com/Longju000)
- [@lorenzodonadio](https://github.com/lorenzodonadio)
- [@lpinca](https://github.com/lpinca)
- [@lqqyt2423](https://github.com/lqqyt2423)
- [@lucasmichot](https://github.com/lucasmichot)
- [@LukasKastern](https://github.com/LukasKastern)
- [@lukeingalls](https://github.com/lukeingalls)
- [@m1212e](https://github.com/m1212e)
- [@malthe](https://github.com/malthe)
- [@markusn](https://github.com/markusn)
- [@Marukome0743](https://github.com/Marukome0743)
- [@marvinruder](https://github.com/marvinruder)
- [@maschwenk](https://github.com/maschwenk)
- [@MasterGordon](https://github.com/MasterGordon)
- [@masterujjval](https://github.com/masterujjval)
- [@mathiasrw](https://github.com/mathiasrw)
- [@MatricalDefunkt](https://github.com/MatricalDefunkt)
- [@matthewyu01](https://github.com/matthewyu01)
- [@maxmilton](https://github.com/maxmilton)
- [@meck93](https://github.com/meck93)
- [@mi4uu](https://github.com/mi4uu)
- [@miccou](https://github.com/miccou)
- [@mimikun](https://github.com/mimikun)
- [@mkayander](https://github.com/mkayander)
- [@mkossoris](https://github.com/mkossoris)
- [@morisk](https://github.com/morisk)
- [@mountainash](https://github.com/mountainash)
- [@moznion](https://github.com/moznion)
- [@mroyme](https://github.com/mroyme)
- [@MuhibAhmed](https://github.com/MuhibAhmed)
- [@nangchan](https://github.com/nangchan)
- [@nathanhammond](https://github.com/nathanhammond)
- [@nazeelashraf](https://github.com/nazeelashraf)
- [@nellfs](https://github.com/nellfs)
- [@nil1511](https://github.com/nil1511)
- [@nithinkjoy-tech](https://github.com/nithinkjoy-tech)
- [@NReilingh](https://github.com/NReilingh)
- [@nshen](https://github.com/nshen)
- [@nullun](https://github.com/nullun)
- [@nxzq](https://github.com/nxzq)
- [@nygmaaa](https://github.com/nygmaaa)
- [@o-az](https://github.com/o-az)
- [@o2sevruk](https://github.com/o2sevruk)
- [@oguimbal](https://github.com/oguimbal)
- [@Osmose](https://github.com/Osmose)
- [@otgerrogla](https://github.com/otgerrogla)
- [@otterDeveloper](https://github.com/otterDeveloper)
- [@owlcode](https://github.com/owlcode)
- [@pacexy](https://github.com/pacexy)
- [@pan93412](https://github.com/pan93412)
- [@Pandapip1](https://github.com/Pandapip1)
- [@panva](https://github.com/panva)
- [@paperless](https://github.com/paperless)
- [@Parzival-3141](https://github.com/Parzival-3141)
- [@PaulaBurgheleaGithub](https://github.com/PaulaBurgheleaGithub)
- [@paulbaumgart](https://github.com/paulbaumgart)
- [@PaulRBerg](https://github.com/PaulRBerg)
- [@ped](https://github.com/ped)
- [@Pedromdsn](https://github.com/Pedromdsn)
- [@perpetualsquid](https://github.com/perpetualsquid)
- [@pesterev](https://github.com/pesterev)
- [@pfgithub](https://github.com/pfgithub)
- [@philolo1](https://github.com/philolo1)
- [@pierre-cm](https://github.com/pierre-cm)
- [@Pierre-Mike](https://github.com/Pierre-Mike)
- [@pnodet](https://github.com/pnodet)
- [@polarathene](https://github.com/polarathene)
- [@PondWader](https://github.com/PondWader)
- [@prabhatexit0](https://github.com/prabhatexit0)
- [@Primexz](https://github.com/Primexz)
- [@qhariN](https://github.com/qhariN)
- [@RaisinTen](https://github.com/RaisinTen)
- [@rajatdua](https://github.com/rajatdua)
- [@RaresAil](https://github.com/RaresAil)
- [@rauny-brandao](https://github.com/rauny-brandao)
- [@rhyzx](https://github.com/rhyzx)
- [@RiskyMH](https://github.com/RiskyMH)
- [@risu729](https://github.com/risu729)
- [@RodrigoDornelles](https://github.com/RodrigoDornelles)
- [@rohanmayya](https://github.com/rohanmayya)
- [@RohitKaushal7](https://github.com/RohitKaushal7)
- [@rtxanson](https://github.com/rtxanson)
- [@ruihe774](https://github.com/ruihe774)
- [@rupurt](https://github.com/rupurt)
- [@ryands17](https://github.com/ryands17)
- [@s-rigaud](https://github.com/s-rigaud)
- [@s0h311](https://github.com/s0h311)
- [@saklani](https://github.com/saklani)
- [@samfundev](https://github.com/samfundev)
- [@samualtnorman](https://github.com/samualtnorman)
- [@sanyamkamat](https://github.com/sanyamkamat)
- [@scotttrinh](https://github.com/scotttrinh)
- [@SeedyROM](https://github.com/SeedyROM)
- [@sequencerr](https://github.com/sequencerr)
- [@sharpobject](https://github.com/sharpobject)
- [@shinichy](https://github.com/shinichy)
- [@silversquirl](https://github.com/silversquirl)
- [@simylein](https://github.com/simylein)
- [@sirenkovladd](https://github.com/sirenkovladd)
- [@sirhypernova](https://github.com/sirhypernova)
- [@sitiom](https://github.com/sitiom)
- [@Slikon](https://github.com/Slikon)
- [@Smoothieewastaken](https://github.com/Smoothieewastaken)
- [@sonyarianto](https://github.com/sonyarianto)
- [@Southpaw1496](https://github.com/Southpaw1496)
- [@spicyzboss](https://github.com/spicyzboss)
- [@sroussey](https://github.com/sroussey)
- [@sstephant](https://github.com/sstephant)
- [@starsep](https://github.com/starsep)
- [@stav](https://github.com/stav)
- [@styfle](https://github.com/styfle)
- [@SukkaW](https://github.com/SukkaW)
- [@sum117](https://github.com/sum117)
- [@techvlad](https://github.com/techvlad)
- [@thapasusheel](https://github.com/thapasusheel)
- [@ThatOneBro](https://github.com/ThatOneBro)
- [@ThatOneCalculator](https://github.com/ThatOneCalculator)
- [@therealrinku](https://github.com/therealrinku)
- [@thunfisch987](https://github.com/thunfisch987)
- [@tikotzky](https://github.com/tikotzky)
- [@tk120404](https://github.com/tk120404)
- [@tobycm](https://github.com/tobycm)
- [@tom-sherman](https://github.com/tom-sherman)
- [@tomredman](https://github.com/tomredman)
- [@toneyzhen](https://github.com/toneyzhen)
- [@toshok](https://github.com/toshok)
- [@traviscooper](https://github.com/traviscooper)
- [@trnxdev](https://github.com/trnxdev)
- [@tsndr](https://github.com/tsndr)
- [@TwanLuttik](https://github.com/TwanLuttik)
- [@twlite](https://github.com/twlite)
- [@VietnamecDevelopment](https://github.com/VietnamecDevelopment)
- [@Vilsol](https://github.com/Vilsol)
- [@vinnichase](https://github.com/vinnichase)
- [@vitalspace](https://github.com/vitalspace)
- [@vitoorgomes](https://github.com/vitoorgomes)
- [@vitordino](https://github.com/vitordino)
- [@vjpr](https://github.com/vjpr)
- [@vladaman](https://github.com/vladaman)
- [@vlechemin](https://github.com/vlechemin)
- [@Voldemat](https://github.com/Voldemat)
- [@vthemelis](https://github.com/vthemelis)
- [@vveisard](https://github.com/vveisard)
- [@wbjohn](https://github.com/wbjohn)
- [@weyert](https://github.com/weyert)
- [@whygee-dev](https://github.com/whygee-dev)
- [@winghouchan](https://github.com/winghouchan)
- [@WingLim](https://github.com/WingLim)
- [@wobsoriano](https://github.com/wobsoriano)
- [@xbjfk](https://github.com/xbjfk)
- [@xHyroM](https://github.com/xHyroM)
- [@ximex](https://github.com/ximex)
- [@xlc](https://github.com/xlc)
- [@xNaCly](https://github.com/xNaCly)
- [@yadav-saurabh](https://github.com/yadav-saurabh)
- [@yamcodes](https://github.com/yamcodes)
- [@Yash-Singh1](https://github.com/Yash-Singh1)
- [@YashoSharma](https://github.com/YashoSharma)
- [@yharaskrik](https://github.com/yharaskrik)
- [@Yonben](https://github.com/Yonben)
- [@yschroe](https://github.com/yschroe)
- [@yukulele](https://github.com/yukulele)
- [@yus-ham](https://github.com/yus-ham)
- [@zack466](https://github.com/zack466)
- [@zenshixd](https://github.com/zenshixd)
- [@zieka](https://github.com/zieka)
- [@zongzi531](https://github.com/zongzi531)
- [@ZTL-UwU](https://github.com/ZTL-UwU)

**Full Changelog**: https://github.com/oven-sh/bun/compare/bun-v1.0.0...bun-v1.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/oven-sh/bun/releases/tag/bun-v1.1.0'>Bun v1.1.0</a>.</em>

**Comment by Zaid-maker** [comment]
Thank u so much for the support of Windows ❤️
